osfoffline/sync/local.py

```diff
@@ -21,11 +22,19 @@ class LocalSyncWorker(ConsolidatedEventHandler, metaclass=Singleton):
 
     def __init__(self):
         super().__init__()
-        self.folder = get_current_user().folder
+        try:
+            user = get_current_user()
+        except NoResultFound:
+            user = None
+            self.folder = None
+
+        if user:
+            self.folder = user.folder
 
         self.observer = Observer()
         self.ignore = threading.Event()
-        self.observer.schedule(self, self.folder, recursive=True)
+        if self.folder:
```

Comment:
That's a good point. For the time being letting the exception go unhandled is the best approach here.